experiments/integrated/fluxnet/met_drivers_FLUXNET.jl

```diff
@@ -181,23 +176,24 @@ radiation = ClimaLand.PrescribedRadiativeFluxes(
 )
 
 # Start and end dates of data in MODIS format
-modis_start_date = "A$(Dates.year(UTC_DATETIME[1]))$(lpad(Dates.dayofyear(UTC_DATETIME[1]), 3, "0"))"
-modis_end_date = "A$(Dates.year(UTC_DATETIME[end]))$(lpad(Dates.dayofyear(UTC_DATETIME[end]), 3, "0"))"
-
-MODIS_LAI = single_col_data_matrix(
-    parse_response(
-        check_response(
-            send_get_subset(
-                "MCD15A2H",
-                modis_start_date,
-                modis_end_date,
-                site_ID,
-                band = "Lai_500m",
-            ),
-        ),
-    ),
+modis_start_date = DateTime(
+    "$(Dates.year(UTC_DATETIME[1]))-$(Dates.month(UTC_DATETIME[1]))-$(Dates.day(UTC_DATETIME[1]))T00:00:00.0",
+)
+modis_end_date = DateTime(
+    "$(Dates.year(UTC_DATETIME[end]))-$(Dates.month(UTC_DATETIME[end]))-$(Dates.day(UTC_DATETIME[end]))T00:00:00.0",
 )
```

Comment:
It would be good to add a short comment explaining that these lines put the simulation start and end date into the format used by the MODIS data